contracts/LightClientBridge.sol

```diff
@@ -160,7 +167,7 @@ contract LightClientBridge {
             //  valid based on the ValidatorRegistry merkle root, ie, confirm that
             //  the randomSignerAddress is from an active validator and is at the correct position
             require(
-                validatorRegistry.checkValidatorInSet(randomValidatorAddresses[i], randomPublicKeyMerkleProofs[i], i),
+                validatorRegistry.checkValidatorInSet(randomValidatorAddresses[i], randomPublicKeyMerkleProofs[i]),
```

Comment:
`i` needs to be verified here as well, I just removed it because the test would fail for me. Please add it again and change the implementation to check the position in the merkle tree.